Model/ConfirmedProcessor.php

```diff
@@ -38,9 +38,14 @@ public function processConfirmedWebhook($params, $order)
         $selectedPaymentMethod = (float)$params['selectedPaymentMethod'];
         $orderPayment = $order->getPayment();
         $orderPayment
-            ->setAdditionalInformation(BalancepayMethod::BALANCEPAY_IS_FINANCED, $isFinanced)
-            ->setAdditionalInformation(BalancepayMethod::
-            BALANCEPAY_SELECTED_PAYMENT_METHOD, $selectedPaymentMethod);
+            ->setAdditionalInformation(BalancepayMethod::BALANCEPAY_IS_FINANCED, $isFinanced);
+        if ($this->balancepayConfig->getIsAuth()) {
+            $orderPayment->setAdditionalInformation(BalancepayMethod::BALANCEPAY_IS_AUTH_CHECKOUT, 'Authorization');
+        } else {
+            $orderPayment->setAdditionalInformation(BalancepayMethod::BALANCEPAY_IS_AUTH_CHECKOUT, 'Sale');
+        }
```

Comment:
```suggestion
        $transactionType = $this->balancepayConfig->getIsAuth() ? 'Authorization' : 'Sale';
        $orderPayment->setAdditionalInformation(BalancepayMethod::BALANCEPAY_IS_AUTH_CHECKOUT, $transactionType);
```

Reply:
What is `BALANCEPAY_IS_AUTH_CHECKOUT`?

Reply:
It is a additional data which we are setting.